app/validation/validator.py

```diff
@@ -515,6 +515,24 @@ def _get_question_variant_fields_sets(variants):
 
             results["number_of_answers"].add(len(results["answer_ids"]))
 
+        # Code to handle comparison of variants which contain a MutuallyExclusive answer type
+        if (
+            len(results["question_types"]) > 1
+            and "MutuallyExclusive" in results["question_types"]
+        ):
+            results["question_types"].remove("MutuallyExclusive")
```

Comment:
This will only remove the element at the first index that matches, so if there are more than one MutuallyExclusive answer N>1 won't be removed. Is that the expected behaviour?

Reply:
I thought as it is a set there would only ever be one Mutually exclusive element?

Reply:
Ah yeah - I assumed it was a list :+1: